Jenkinsfile-test

```diff
@@ -35,15 +35,7 @@ pipeline {
         expression { env.BRANCH_NAME == "master"}
       }
       stages {
-        stage('Tag Release') {
-          steps {
-            sh "git tag ${versionTag}"
-            sshagent(['github-jenkins']) {
-              sh("git push origin ${versionTag}")
-            }
-          }
-        }
-        stage('Deploy to integration') {
+        stage('Publish library') {
```

Comment:
Is the `def versionTag = "v${env.BUILD_NUMBER}"` on line 3 required anymore?

Reply:
Fixed, thanks.